packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -167,6 +168,131 @@ def vip_workload_test(dcos_api_session, container, vip_net, proxy_net, named_vip
+            for proxy_net in [marathon.Network.USER, marathon.Network.BRIDGE, marathon.Network.HOST]]
+
+
+@pytest.mark.slow
+@pytest.mark.skipif(not lb_enabled(), reason='Load Balancer disabled')
+@pytest.mark.parametrize('vip_net,proxy_net', generate_pod_vip_app_permutations())
+def test_pod_vip(dcos_api_session, vip_net: marathon.Network, proxy_net: marathon.Network):
+    """ Same as test_vip but for pods
+        * containers: UCR
+        * networks: USER, HOST
+        * vips: named and unnamed vip
+    """
+    errors = 0
+    tests = setup_pod_vip_workload_tests(dcos_api_session, vip_net, proxy_net)
+    for vip, hosts, cmd, origin_app, proxy_app in tests:
+        log.info("Testing :: VIP: {}, Hosts: {}".format(vip, hosts))
+        log.info("Remote command: {}".format(cmd))
+        proxy_info = dcos_api_session.marathon.get('v2/pods/{}::status'.format(proxy_app['id'])).json()
+        log.info(proxy_info)
+        if proxy_net == marathon.Network.USER:
+            proxy_host = proxy_info['instances'][0]['networks'][0]['addresses'][0]
+            proxy_port = proxy_app['containers'][0]['endpoints'][0]['containerPort']
+        else:
+            proxy_host = proxy_info['instances'][0]['agentHostname']
+            proxy_port = proxy_info['instances'][0]['containers'][0]['endpoints'][0]['allocatedHostPort']
+        try:
+            assert ensure_routable(cmd, proxy_host, proxy_port)['test_uuid'] == \
+                origin_app['containers'][0]['environment']['DCOS_TEST_UUID']
+        except Exception as ex:
+            log.error('Exception: {}'.format(ex))
+            errors = errors + 1
+        finally:
+            log.info('Purging application: {}'.format(origin_app['id']))
+            dcos_api_session.marathon.delete('v2/pods/{}'.format(origin_app['id']))
+            log.info('Purging application: {}'.format(proxy_app['id']))
+            dcos_api_session.marathon.delete('v2/pods/{}'.format(proxy_app['id']))
+    assert errors == 0
+
+
+def setup_pod_vip_workload_tests(dcos_api_session, vip_net, proxy_net):
+    if len(dcos_api_session.all_slaves) < 2 and vip_net is marathon.Network.BRIDGE:
+        return []
+    same_hosts = [True, False] if vip_net is not marathon.Network.BRIDGE else [False]
+    tests = [pod_vip_workload_test(dcos_api_session, vip_net, proxy_net, named_vip, same_host)
+             for named_vip in [True, False]
+             for same_host in same_hosts]
+    for vip, hosts, cmd, origin_app, proxy_app in tests:
+        log.info('Starting apps :: VIP: {}, Hosts: {}'.format(vip, hosts))
+        log.info("Origin app: {}".format(origin_app))
+        dcos_api_session.marathon.post('v2/pods', json=origin_app).raise_for_status()
+        log.info("Proxy app: {}".format(proxy_app))
+        dcos_api_session.marathon.post('v2/pods', json=proxy_app).raise_for_status()
+    for vip, hosts, cmd, origin_app, proxy_app in tests:
+        log.info("Deploying apps :: VIP: {}, Hosts: {}".format(vip, hosts))
```

Comment:
Can we change this debug log to "Waiting for app to be healthy" to be more explicit in our debug logs?